docs/user/ppl/functions/datetime.rst

```diff
@@ -516,58 +516,6 @@ Example::
     +------------------------------------------------------+-----------------------------------------------------------------------+
 
 
-DATETIME
```

Comment:
This function still exists and was not in scope to remove for this PR (even though the name of the function was `datetime`.

Reply:
That's a good point, I removed it because the datetime type was taken out but I suppose I should have just edited the function to instead return a timestamp value

Reply:
documentation added back in in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/1aec4f9ee0ca2e447f3a4b9cdff2169a98c76fcb